ccmbioinfo/lfs
# CARDS-1716: As a patient user, I can authenticate without a token

Known improvements:
- Add restricted tokens so tokenless authentication users have less data access (patient information form, visit information form, submitted surveys)

Testing:
- Create patients with a DOB and MRN or HC
- Create visits with various values for surveys, visit time (future or past) and status ("cancelled" or anything else)

Only visits with a survey answer, a visit time in the future and a status other than "cancelled" should be shown.

On the Proms homepage, enter a DOB and MRN or HC. If this is not a valid patient or if the patient doesn't have any valid visits, an "invalid credentials" error should be shown. There should be no way for a user to differentiate between an invalid patient and valid patient with no valid visits.
If the patient is valid and they have 1 valid upcoming visit, the login process should proceed as before (terms of use if required, then form list and submission).
If the patient is valid and they have >1 valid upcoming visit, they should be presented with a list of "survey" answers for each upcoming visit. They should then be able to select one of these, and then proceed with completing that visit.
